examples/k8s/.redun/redun.ini

```diff
@@ -0,0 +1,15 @@
+# redun configuration.
+
+[backend]
+db_uri = sqlite:///redun.db
+
+[executors.default]
+type = local
+max_workers = 20
+
+[executors.k8s]
+type = k8s
+image = YOUR_AWS_ACCOUNT.dkr.ecr.YOUR_AWS_REGION.amazonaws.com/redun_example
+queue = YOUR_QUEUE
+s3_scratch = s3://YOUR_BUCKET/redun/
+min_array_size = 100
```

Comment:
I'm currently disabling the job array here (by making the min usually high), because I had some trouble with it during local testing. I'll undo this once I get array jobs working.

Reply:
Matt, #52 should fix the job array feature, I don't think it's been merged yet.